security/src/main/java/io/micronaut/security/utils/LoggingUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.security.utils;
+
+import org.slf4j.Logger;
+
+/**
+ * Utility methods to avoid verbosity of logging statements. Mostly used to help reduce the Cognitive Complexity of
+ * some methods.
+ */
+public final class LoggingUtils {
```

Comment:
should this class be part of core?

Reply:
Potentially. @graemerocher thoughts?

Still, I would like to move forward with it belonging here. We can move it if/when it's part of core.

I'm also annotating it with `@Internal` just in case.

Reply:
Annotate with `@Internal` in case we move it to core soon.

Reply:
I am not sure if you added your comment before mine, but I just said that I'll do it. Actually it's committed and pushed now.

Reply:
We were thinking the same. Thanks!